bin/release.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
I've started using:
```suggestion
set -euo pipefail
```
since reading an article on bash scripting lately. It has been helpful to me.

Reply:
I've tried to use `set -euo pipefail` in the past and I think it tends to work well for scripts where you don't want to manually catch errors and send meaningful messages back to the user, ie. it just bombs out immediately (maybe automation or non-interactive scripts?). It also crashes sometimes for things you wouldn't expect to be a fatal error...

That said, I haven't tried it for this particular script, so it might be worth exploring.

Reply:
I tested it and this worked fine, thanks!